opendistro-elasticsearch-index-management-kibana.release-notes.md

```diff
@@ -1,3 +1,7 @@
+## Version 1.7.0.0
+
+patch release to be compatible with other plugins
```

Comment:
Change to: Adds support for ODFE 1.7.0

Reply:
modified, and organize the release notes as Alolita said